airbyte-workers/src/main/java/io/airbyte/workers/internal/DefaultAirbyteStreamFactory.java

```diff
@@ -94,9 +94,9 @@ private void internalLog(final AirbyteLogMessage logMessage) {
     switch (logMessage.getLevel()) {
       case FATAL, ERROR -> logger.error(logMessage.getMessage());
       case WARN -> logger.warn(logMessage.getMessage());
-      case INFO -> logger.info(logMessage.getMessage());
       case DEBUG -> logger.debug(logMessage.getMessage());
       case TRACE -> logger.trace(logMessage.getMessage());
+      default -> logger.info(logMessage.getMessage());
```

Comment:
opinions on this default? (similar to above)

Reply:
This seems fine.